flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/sink/operator/TiDBWriteOperator.java

```diff
@@ -74,9 +68,6 @@ public TiDBWriteOperator(
   public void open() throws Exception {
     this.session = ClientSession.create(new ClientConfig(properties));
     this.tiTableInfo = session.getTableMust(databaseName, tableName);
-    this.rowIDAllocator =
-        new DynamicRowIDAllocator(
-            session, databaseName, tableName, sinkOptions.getRowIdAllocatorStep());
```

Comment:
why move it into Subclass？

Reply:
DynamicRowIDAllocator is associated with Transaction.
For GlobalWriteOperator, it needs to be a member variable since there is only a transaction.
For MinibatchWriteOperator, it needs to be a temp variable in method flushrows() since there are various transactions.